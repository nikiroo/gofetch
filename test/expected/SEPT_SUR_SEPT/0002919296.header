0"Rien n%27a été convenu": Courtois évoque la concurrence avec Navas	null/SEPT_SUR_SEPT/0002919296		70
i  [Football Etranger] 26-09-2018 16:49
i
i    Thibaut  Courtois  doit  faire  face à la féroce concurrence de
i    Keylor  Navas  depuis  son  arrivée au Real Madrid. Comme l%27a
i    confirmé le Diable, Julen ...
i
