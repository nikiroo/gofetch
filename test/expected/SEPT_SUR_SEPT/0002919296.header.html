<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h2><a href='0002919296.html'>"Rien n%27a été convenu": Courtois évoque la concurrence avec Navas</a></h2>
	<div class='details'>([Football Etranger] 26-09-2018 16:49)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		Thibaut Courtois doit faire face à la féroce concurrence de Keylor Navas depuis son arrivée au Real Madrid. Comme l%27a confirmé le Diable, Julen ...
	</div>
<hr/>
</div>
</body>
